app/views/user_mailer/_request_expired.html.erb

```diff
@@ -0,0 +1,3 @@
+<p>Your <%= link_to 'reservation request', reservation_url(@reservation, only_path: false) %> has expired.</p>
+
+<p> You can request the equipment again <%= link_to 'here', equipment_model_url(@reservation.equipment_model), only_path: false %>.</p>
```

Comment:
I think that option hash needs to be *inside* that parentheses :stuck_out_tongue: